TinyMCE.Blazor/wwwroot/TinyMce.Blazor.lib.module.js

```diff
@@ -85,6 +85,11 @@ const chunkMap = (() => {
 })();
 
 window.tinymceBlazorWrapper = {
+  insertContent: (id, content, args) => {
+    console.log('insetting content', content, args);
```

Comment:
Do we need the console.log? Also should probably be `inserting` instead of `insetting`.